src/opnsense/mvc/app/library/OPNsense/Backup/Nextcloud.php

```diff
@@ -128,13 +135,16 @@ function backup()
             $username = (string)$nextcloud->user;
             $password = (string)$nextcloud->password;
             $backupdir = (string)$nextcloud->backupdir;
+            $crypto_password = (string)$nextcloud->password_encryption;
             $hostname = $config->system->hostname . '.' .$config->system->domain;
-            $configname = 'config-' . $hostname . '-' .  date("Y-m-d_h:m:s") . '.xml';
+            $configname = 'config-' . $hostname . '-' .  date("Y-m-d_H:i:s") . '.xml';
```

Comment:
Or have we not fixed the time everywhere yet?

Reply:
my branch was probably too old - however git should not show it if it is equal on both branches...

Reply:
hm, no github is bugging out here in this PR making it useless for opnsense-patch